resources/js/store/modules/teams/actions.js

```diff
@@ -0,0 +1,154 @@
+import Vue from 'vue'
+import i18n from '../../../i18n'
+import router from "../../../routes";
+
+export const actions = {
+
+    /**
+     * The user wants to create a new team
+     */
+    async CREATE_NEW_TEAM (context, payload)
+    {
+        const title = i18n.t('notifications.success');
+        const body  = i18n.t('teams.created');
+
+        const error_title = i18n.t('notifications.error');
+        const error_body  = i18n.t('teams.create.max-created');
+
+        await axios.post('/teams/create', {
+            name: payload.name,
+            identifier: payload.identifier,
+            teamType: payload.teamType
+        })
+        .then(response => {
+            console.log('create_new_team', response);
+
+            if (response.data.success)
+            {
+                // show notification
+                Vue.$vToastify.success({
+                    title,
+                    body,
+                    position: 'top-right'
+                });
+
+                // add team to teams array
+                // context.commit('usersTeams', response.data.team);
+
+                // If user does not have an active team, assign
+                if (! context.rootState.user.user.team_id) {
```

Comment:
Should not have space between ! and `context.rootState.user.user.team_id`

Reply:
We always have space after operator

Reply:
Yeah but I think we should not do that with not operator. Check this [link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_NOT), I follow this style